.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,36 @@
+name: Documentation bug
+description: Raise an issue related to content.
+title: "🐞 Documentation Bug or documentation feature: "
```

Comment:
I feel like the emoji should be consistent.
Don't get me wrong, I do like emojis because they bring life to the project, but we don't have any emojis so far in the Tracim repo.
So either add it on other templates or remove this one :confused:

Reply:
Sorry, it was a copy/paste from scaleway template 😂